src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int MAX_NUMBER_OF_BALLS = 3;
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < MAX_NUMBER_OF_BALLS; i++) {
+            System.out.println(lottery.getRandomBall().toString());
```

Comment:
There is no need to explicitly call toString() in the println method, as it is called on passed objects under the hood